py-polars/polars/dataframe/frame.py

```diff
@@ -727,7 +727,7 @@ def _read_csv(
             if dtype_slice is not None:
                 raise ValueError(
                     "cannot use glob patterns and unnamed dtypes as `dtypes` argument"
-                    "\n\nUse `dtypes`: Mapping[str, Type[DataType]"
+                    "\n\nUse `dtypes`: Mapping[str, Type[DataType]]."
```

Comment:
Good catch on the `]` but this doesn't need a period. When listing a single thing after a colon this just makes things less clear.